package/MDAnalysis/analysis/psa.py

```diff
@@ -573,7 +574,15 @@ def dist_mat_to_vec(N, i, j):
     :Returns:
       int, index (of the matrix element) in the corresponding distance vector
     """
-    if i > N or j > N:
+
+    if not (isinstance(N, int) or isinstance(i, int) or isinstance(j, int)):
```

Comment:
Does this still work with numpy integers? So you pass `np.int32(3)` or `np.int64(3)` to the function?